File: coco/src/main/java/coco/ep/m7i2/Exp.java
package coco.ep.m7i2;

public interface Exp<FT> extends coco.ep.m7.Exp<FT>, coco.ep.i1.Exp<FT>, Factory<FT> {

    default Boolean isPower(coco.ep.Exp<FT> left, coco.ep.Exp<FT> right) { return false; }

}

File: coco_csharp/CoCo/EP/M7I2/Exp.cs
namespace EP.M7I2 {
    public interface Exp<FT> : Factory<FT>, EP.M7.Exp<FT>, EP.I1.Exp<FT> {
        bool isPower(EP.Exp<FT> left, EP.Exp<FT> right) { return false; }
    }
}